tests/framework/microvm.py

```diff
@@ -282,6 +284,10 @@ def kill(self):
         for monitor in self.monitors:
             monitor.stop()
 
+        # Kill all background SSH connections
+        for connection in self._connections:
+            connection.close()
+
```

Comment:
There may be a way of introspecting the `ssh_iface` cache. Or we could do `self.ssh_iface.cache_clear()` and close the connection on `SSHConnection.__del__`  though I don't like desctructors that much..

Reply:
Nvm, just read the commit body about @lru_cache treating.. Though it may be worth removing the default iface_idx = 0. I don't think anything depends on it?

Reply:
Yeah, I tried inspecting the lru_cache and it didn't seem to provide a way to get access to the cached objects. Can probably remove the default, but I guess now that test_net.py is updated its also not causing any problems anymore haha

Reply:
Btw destructors also wouldn't work because the connections need to be closed before the microvm is killed, because further down in microvm.kill() there's the check about "something using the jailer id is still alive"